### ISSUE TITLE
Instant search on category page 

### ISSUE THREAD

**Issue by rehamwael** [algolia/algoliasearch-magento#681]
We have set "Enable Auto-completion Menu" and now our Top Search works as an instant search.

But the problem is that when I visit a category, Top Search searches only inside of that category pages doesn't work the top search as instant search it's give me an empty output only on the category pages ?
If I set replace category page to yes its append the new category after my old category page 
any help please 
I know that i ask a lot :(

Thanks on Advance

**Comment by JanPetr** [comment]
Hello @rehamwael,

thank you for the report. It looks like an issue in the extension. I will investigate and come back to you. Sorry for the delay.

**Comment by JanPetr** [comment]
Hello @rehamwael,

I finally got to your issue. We found out where the issue is and currently we're working on a fix.
The approach we're taking here is to display current category in a current refinements widget and give a user a possibility to remove the category refinement, so the search can be performed either on the whole catalog or within the selected category.

Hope it will solve the issue. I will let you know once it's implemented.
